store = {}
store['timestamp']=1621445980
store['cmdline']=['/home/kell4989/git/active_learning_redux/batchbald_redux/uniform_target_ood_experiment.py', '--id=3']
store['commit']='fe39332a0e442ffbcad51ed35b57d7e83aca3bb9'
store['github_url']='fe39332a0e442ffbcad51ed35b57d7e83aca3bb9'
store['experiment']='/home/kell4989/git/active_learning_redux/batchbald_redux/uniform_target_ood_experiment.py'
store['job_id']=3
store['worker_id']=3
store['num_workers']=20
store['config']={'seed': 1238, 'acquisition_size': 5, 'max_training_set': 200, 'num_pool_samples': 100, 'num_validation_samples': 20, 'num_training_samples': 1, 'num_patience_epochs': 3, 'max_training_epochs': 30, 'training_batch_size': 64, 'device': 'cuda', 'validation_set_size': 1024, 'evaluation_set_size': 10000, 'validation_split_random_state': 0, 'initial_training_set_size': 20, 'min_samples_per_epoch': 5056, 'mnist_repetitions': 1, 'ood_fmnist_repetitions': 1, 'add_dataset_noise': False, 'acquisition_function': 'batchbald_redux.acquisition_functions.BatchBALD', 'train_eval_model': 'batchbald_redux.train_eval_model.TrainSelfDistillationEvalModel', 'model_optimizer_factory': 'batchbald_redux.models.MnistOptimizerFactory', 'acquisition_function_args': None, 'temperature': 8}
store['log']={}
store['exception']='Traceback (most recent call last):\n  File "/home/kell4989/git/active_learning_redux/batchbald_redux/uniform_target_ood_experiment.py", line 319, in <module>\n    config.run(store=store)\n  File "/home/kell4989/git/active_learning_redux/batchbald_redux/uniform_target_ood_experiment.py", line 175, in run\n    data = self.load_experiment_data()\n  File "/home/kell4989/git/active_learning_redux/batchbald_redux/uniform_target_ood_experiment.py", line 92, in load_experiment_data\n    ood_dataset = FastMNIST("data", train=True, download=True, device=self.device)\n  File "/home/kell4989/git/active_learning_redux/batchbald_redux/fast_mnist.py", line 19, in __init__\n    super().__init__(*args, **kwargs)\n  File "/tmp/conda_envs/job-1582787/lib/python3.8/site-packages/torchvision/datasets/mnist.py", line 95, in __init__\n    self.data, self.targets = torch.load(os.path.join(self.processed_folder, data_file))\n  File "/tmp/conda_envs/job-1582787/lib/python3.8/site-packages/torch/serialization.py", line 592, in load\n    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)\n  File "/tmp/conda_envs/job-1582787/lib/python3.8/site-packages/torch/serialization.py", line 851, in _load\n    result = unpickler.load()\n  File "/tmp/conda_envs/job-1582787/lib/python3.8/site-packages/torch/serialization.py", line 843, in persistent_load\n    load_tensor(data_type, size, key, _maybe_decode_ascii(location))\n  File "/tmp/conda_envs/job-1582787/lib/python3.8/site-packages/torch/serialization.py", line 831, in load_tensor\n    storage = zip_file.get_storage_from_record(name, size, dtype).storage()\nRuntimeError: [enforce fail at inline_container.cc:222] . file not found: archive/data/94851772227632\n'
